test-cypress/integration/setup/jdbc/jdbc.spec.js

```diff
@@ -80,7 +80,7 @@ describe('JDBC configuration', () => {
         JdbcSteps.clickOnEditButton();
         // change the query,
         YasqeSteps.clearEditor();
-        YasqeSteps.writeInEditor(EDIT_QUERY);
+        YasqeSteps.pasteQuery(EDIT_QUERY);
```

Comment:
Revert these. Pasting the query doesn't trigger change and editor remains pristine which makes the tests to report false positives